scan/src/components/TimeAgos.re

```diff
@@ -1,3 +1,5 @@
+open Css;
```

Comment:
Do not open Css on a file level. It will hurt you as Css contains so many common names. Either declare `Styles` module and declare styles in there, or use `Css.` everywhere. Your choice.

Reply:
You're right